patches/aegir+42.2.2.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/node_modules/aegir/src/build/index.js b/node_modules/aegir/src/build/index.js
```

Comment:
created issue in ipfs/aegir: https://github.com/ipfs/aegir/issues/1481.

will remove.